### DISCUSSION TITLE
How to construct JS objects - where is the `new` operator?

### DISCUSSION THREAD

**Discussion by scally** [ruby/ruby.wasm#165] [Q&A]
How do you call `new` on Javascript objects? I haven't found a workaround yet that will let me do this. I can call the object constructor but it will then tell me that I need to call the `new` operator:

for example:
```
observer = JS.global.MutationObserver do
  puts 'here'
end
```
results in 
```
TypeError: Constructor requires 'new' operator (JS::Error)
```

**Comment by kateinoigakukun** [comment]
Currently, there is no way to call `new` in JavaScript. Let me put it in TODOs. (Sorry for slow reply, I missed your post :bow:)
